modules/login/php/login.class.inc

```diff
@@ -57,6 +57,9 @@ class Login extends \NDB_Page
 
         $this->tpl_data['study_title'] = $config->getSetting('title');
 
+        $factory = \NDB_Factory::singleton();
+        $this->tpl_data['baseurl'] = $factory->settings()->getBaseURL();
```

Comment:
Apparently baseurl is being undefined for smarty login unless this is n ow set in the login module. Note: This maybe is unnecessary or needed for when the Login react PR gets merged